### ISSUE TITLE
Instrumental smearing not set automatically based on data (Trac #638)

### ISSUE THREAD

**Issue by ajj** [SasView/sasview#770]
The type of smearing ("Use dQ Data" & slit/pinhole) used to be set automatically based on the data being sent to fitting.

This is no longer the case, and apparently wasn't in 3.1.2 either. It should be.

Migrated from http://trac.sasview.org/ticket/638
```json
{
    "status": "closed",
    "changetime": "2016-10-08T18:18:24",
    "_ts": "2016-10-08 18:18:24.592759+00:00",
    "description": "The type of smearing (\"Use dQ Data\" & slit/pinhole) used to be set automatically based on the data being sent to fitting.\n\nThis is no longer the case, and apparently wasn't in 3.1.2 either. It should be.",
    "reporter": "ajj",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2016-08-30T12:09:26",
    "component": "SasView",
    "summary": "Instrumental smearing not set automatically based on data",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "mathieu",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2016/09/03 01:57:09`: **butler** commented:

Or 3.1.1 I found out today

**Comment by ajj** [comment]
Trac update at `2016/09/06 10:14:56`: **ajj** changed milestone from "SasView 4.0.0" to "SasView 4.1.0"

**Comment by mdoucet** [comment]
Trac update at `2016/10/08 18:18:24`: 
* **mathieu** commented:

In changeset b4818d94e15995eb79ffc8596cc2f32748f2dcfe:

```text
#!CommitTicketReference repository="sasview" revision="b4818d94e15995eb79ffc8596cc2f32748f2dcfe"
By default, use the smearing info in the data file. Fixes #770
```


* **mathieu** changed owner from "" to "mathieu"
* **mathieu** changed resolution from "" to "fixed"
* **mathieu** changed status from "new" to "closed"
